config/locales/application_form.yml

```diff
@@ -304,6 +304,10 @@ en:
         - (should never appear)
         - First referee
         - Second referee
+        - Third referee
+        - Fourth referee
+        - Fifth referee
+        - Sixth referee
```

Comment:
Is this logic assumes that a candidate would need to add a new request "a maximum of 6 times"?

Reply:
Yep! I'll make a note of this to improve it.